test/integration/patientCheckIn.js

```diff
@@ -37,4 +39,19 @@ describe('(/patients/:uuid/visits) Patient Check In', () => {
         expect(result.body).to.have.keys('uuid');
       });
   });
+
+
+  it('GET patients/:uuid/invoices/latest :The latest bill should show the most recent bill of a patient', () => {
+    const LIMIT = 1;
+    return agent.get(`/patients/${debtorUuid}/invoices/latest`)
+      .then(function (result) {
+        expect(result).to.have.status(200);
+
+        expect(result.body[0]).to.have.keys('uid', 'reference', 'credit', 'debit', 
```

Comment:
To make this test stronger, check that the `debtor_uuid` is the same as the debtorUuid requested.